index.html

```diff
@@ -1,13 +1,83 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="utf-8" />
-
-    <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Kent's Portfolio Page</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <nav>
+      <div class="navbar">
```

Comment:
Do we need the div? nav is a block element already

Reply:
alright noted. will remove the div.